From e2d2ace9fcb1313aac1e71369dcfc3c26923d667 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 26 Jun 2024 09:58:25 -0700
Subject: [PATCH] Disable -march=native

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 configure.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.in b/configure.in
index 283cca8..2c38098 100644
--- a/configure.in
+++ b/configure.in
@@ -215,10 +215,10 @@ fi
 if test "$enable_opt" != "no"
 then
   printf 'checking for CPU optimization availability... '
-  if printf 'int main(void) {return 0;}' | $CC -xc -march=native -o config.tmp - >config.tmp 2>&1
+  if printf 'int main(void) {return 0;}' | $CC -xc -o config.tmp - >config.tmp 2>&1
   then
-    MYCFLAGS="-march=native $MYCFLAGS"
-    MYCXXFLAGS="-march=native $MYCXXFLAGS"
+    MYCFLAGS="$MYCFLAGS"
+    MYCXXFLAGS="$MYCXXFLAGS"
     printf 'yes\n'
   else
     printf 'no\n'
-- 
2.45.1

